### PR TITLE
feat: VS 2022 Setup Tests

### PR DESCRIPTION
**Pull request by aniruddhadave**
*Description of changes:*
Add VS 2022 Setup Tests

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tst/PortingAssistantExtensionUITests/ControlsTest.cs [hunk 1]
```diff
@@ -27,6 +27,8 @@ namespace PortingAssistantExtensionUITests
     [TestClass]
     public class ControlsTest : VisualStudioSession 
     {
+        public ControlsTest() : base(@"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\devenv.exe") { }
```
Comment (huawenm) [inline tst/PortingAssistantExtensionUITests/ControlsTest.cs:30]: try to check if the file exist before run.

Comment (cslong) [reply tst/PortingAssistantExtensionUITests/ControlsTest.cs:31]: Another thing that might be good to consider is community and enterprise? Not sure if we can encapsulate all that logic together?

Comment (aniruddhadave) [reply tst/PortingAssistantExtensionUITests/ControlsTest.cs:31]: The check if the file exists has been added to the TestInitialization. 
Regarding the type of installation, it should be uniform across test machines/codebuild or we'll need to have a logic to find the correct VS installation as well as preference if multiple versions exist on the machine before running the tests.

**Review by cslong — APPROVED** [review]
lgtm

**Review by saikiranakula-amzn — DISMISSED** [review]

**Review by huawenm — APPROVED** [review]

**Review by huawenm — APPROVED** [review]

**Pull request merged**